modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/snapshot/SnapshotManagerComponent.java

```diff
@@ -116,11 +115,11 @@ public DistributionSnapshot getRuntimeSnapshot() {
 
     @Override
     public DistributionSnapshot getSnapshot(String key, CoreSession session) {
-        if (key == null || RUNTIME.equals(key) || RUNTIME_ADM.equals(key)) {
+        if (key == null || RuntimeSnapshot.LIVE_ALIASES.contains(key)) {
```

Comment:
Shouldn't we remove the `key == null` constraint? Because we throw an exception if the site mode is not enabled which change a bit the behavior of this method.

Reply:
I'm ok with that :+1: , this use case makes things a bit too complicated indeed